### PR TITLE
[dashboard] name org in create dialog

### PR DESCRIPTION
**Pull request by svenefftinge**
## Description

Screenshot 2023-05-22 at 17 14 31

## Related Issue(s)

Fixes WEB-185

#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - se-create-10dad6f0b2</li>
	<li><b>🔗 URL</b> - <a href="https://se-create-10dad6f0b2.preview.gitpod-dev.com/workspaces" target="_blank">se-create-10dad6f0b2.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
</ul>

## Build Options:

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer Options</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [x] with-ws-manager-mk2
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

/hold

### CODE REVIEW

**Comment by easyCZ** [comment]
Screenshot 2023-05-22 at 16 55 47

For personal orgs, this is weird

File: components/dashboard/src/workspaces/CreateWorkspacePage.tsx [hunk 1]
```diff
@@ -353,7 +353,8 @@ export function CreateWorkspacePage() {
             <div className="flex flex-col max-h-screen max-w-lg mx-auto items-center w-full">
                 <Heading1>New Workspace</Heading1>
```
Comment (gtsiolis) [inline components/dashboard/src/workspaces/CreateWorkspacePage.tsx:354]: Cross-posting for visibility from Figma some [relevant discussion](https://www.figma.com/file/MLJUI2WQ3ZSZvd1dGhrfbu?node-id=2194:2911#450684931) (internal) about heading spacing here.

> This (**heading padding**) was an after effect from https://github.com/gitpod-io/gitpod/pull/17405/files#diff-36bccdd9dc3633e37674f9a58735d9a42981797a0d200ffc5bcaaf79c3b999d3L22.

> See relevant discussion in https://gitpod.slack.com/archives/C02EN94AEPL/p1683057656072399 (internal).

> The change was driven from the Dedicated admin onboarding but affected all h1 headings across the dashboard.

> Some space between the h1 and the description below would be nice.

> 

> I'm to blame for removing the space between in https://github.com/gitpod-io/gitpod/pull/17553 but having too much space can also make the element feel visually unbalanced.

> See also relevant discussion in https://gitpod.slack.com/archives/C01KPEPQYE6/p1683652843912329 (internal).

File: components/dashboard/src/workspaces/CreateWorkspacePage.tsx [hunk 2]
```diff
@@ -352,8 +352,9 @@ export function CreateWorkspacePage() {
         <div className="flex flex-col mt-32 mx-auto ">
             <div className="flex flex-col max-h-screen max-w-lg mx-auto items-center w-full">
                 <Heading1>New Workspace</Heading1>
-                <div className="text-gray-500 text-center text-base">
-                    Start a new workspace with the following options.
+                <div className="text-gray-500 text-center text-base mx-6">
+                    Start a workspace with the following options in organization{" "}
+                    <span className="font-semibold">{currentOrg?.name}</span>.
```
Comment (gtsiolis) [inline components/dashboard/src/workspaces/CreateWorkspacePage.tsx:357]: **suggestion**: Minor restructure[^1][^2] of the subheading. Nice catch[[1](https://github.com/gitpod-io/gitpod/pull/17701#issuecomment-1557369345)], @easyCZ! ⚾ 

Frame 1471

[^1]: Using a new _Create_ verb for the first time. Alternatively, we could stick with _Start_.
[^2]: Dropping the options reference which is repeated below the _Continue_ action.

**Review by gtsiolis — APPROVED** [review]
Shippable! 🚢

**Comment by svenefftinge** [comment]
looks like this now
Screenshot 2023-05-22 at 17 29 55

/unhold

**Pull request merged**